src/auth/auth.module.ts

```diff
@@ -1,15 +1,15 @@
 import { Module } from '@nestjs/common';
 import { JwtModule } from '@nestjs/jwt';
 import { PassportModule } from '@nestjs/passport';
-import { UsersModule } from 'src/users/users.module';
+import { UserModule } from 'src/users/users.module';
```

Comment:
Naming Convention: We use plural names for modules, i.e. productsModule, categoriesModule etc.
So, i believe this should be usersModule